agent/ebs/watcher.go

```diff
@@ -125,29 +214,6 @@ func (w *EBSWatcher) HandleEBSResourceAttachment(ebs *apiebs.ResourceAttachment)
 		})
 	}
 
-	// start EBS CSI Driver Managed Daemon
-	// We want to avoid creating a new CSI driver task if there's already one that's not been stopped.
-	if runningCsiTask := w.taskEngine.GetDaemonTask(md.EbsCsiDriver); runningCsiTask != nil && !runningCsiTask.GetKnownStatus().Terminal() {
```

Comment:
nit: since we've moved the CSI creation to now only be created if we have found an EBS volume at every tick, we probably should update the comments about this

Reply:
These comments are deleted. Do you mean we should make the description of `daemonRunning` method more detailed? 
```go
// Checks if EBS Daemon Task is running and starts a new one it if it's not.
func (w *EBSWatcher) daemonRunning() bool {
```

Reply:
oh er sorry I was referring to line 199

Reply:
Ah right. I missed that. Will update. Thanks!

Reply:
Updated in https://github.com/aws/amazon-ecs-agent/pull/3984/files/c5b6385c0d2aa96c14aa81f88c4c0f4c5cbc86ff..3d68cadba4c9dede99e56c0600199cd5daad3f85

Reply:
thanks!